<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>651</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QDockWidget" name="RGBDock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>245</width>
      <height>475</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>245</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>524287</height>
     </size>
    </property>
    <property name="windowTitle">
     <string>RGB</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_4">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="rLCD">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>3</number>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Dec</enum>
            </property>
            <property name="value" stdset="0">
             <double>255.000000000000000</double>
            </property>
            <property name="intValue" stdset="0">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="rSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>132</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="sliderPosition">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="gLCD">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="digitCount">
             <number>3</number>
            </property>
            <property name="intValue" stdset="0">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="gSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>132</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="sliderPosition">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="bLCD">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="digitCount">
             <number>3</number>
            </property>
            <property name="intValue" stdset="0">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="bSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>132</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="sliderPosition">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Point Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="sizeLCD">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="digitCount">
             <number>1</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Filled</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="pSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>132</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenFile</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>rSliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>gSliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>bSliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>pSliderChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>rSliderChanged()</slot>
  <slot>gSliderChanged()</slot>
  <slot>bSliderChanged()</slot>
  <slot>pSliderChanged()</slot>
 </slots>
</ui>
